mr/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputFormat.java

```diff
@@ -66,7 +68,7 @@ public void checkOutputSpecs(FileSystem ignored, JobConf job) {
 
   private static HiveIcebergRecordWriter writer(JobConf jc) {
     TaskAttemptID taskAttemptID = TezUtil.taskAttemptWrapper(jc);
-    Table table = HiveIcebergStorageHandler.table(jc);
+    Table table = HiveIcebergStorageHandler.table(jc, jc.get(hive_metastoreConstants.META_TABLE_NAME));
```

Comment:
At this point, do we know for which output table we should be creating the writer for? I.e. if we have multiple output tables, are we sure that the `name` key in the global conf will contain the correct table name?

Reply:
It gets the config from the `FileSinkOperator` which has its own config for every target table

Reply:
I see, thanks. Can you add this as a one-liner comment above? I think it might help understanding the logic for future maintainers who might be less familiar with the FSO

Reply:
Sure. Done!